### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by BunradarCH**

### CODE REVIEW

File: lab-exercise-1/BunradarFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+  public class BunradarFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (ACitronella) [inline lab-exercise-1/BunradarFeature.java:3]: nice feature

Comment (TreephobKho) [inline lab-exercise-1/BunradarFeature.java:3]: Please include this part.

Comment (Pakin-aw) [inline lab-exercise-1/BunradarFeature.java:3]: Nice comment.

File: lab-exercise-1/BunradarFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+      }
```
Comment (sorrymaker1130) [inline lab-exercise-1/BunradarFeature.java:4]: Good night

File: lab-exercise-1/BunradarFeature.java [hunk 3]
```diff
@@ -1,5 +1,7 @@
   public class BunradarFeature {
       public void feature() {
           // this is a new feature
+          int x = 0;
+          x = x+1;
```
Comment (BunradarCH) [inline lab-exercise-1/BunradarFeature.java:5]: new fixed for two lines added

File: lab-exercise-1/BunradarFeature.java [hunk 4]
```diff
@@ -1,5 +1,7 @@
   public class BunradarFeature {
       public void feature() {
           // this is a new feature
+          int x = 0;
```
Comment (TreephobKho) [inline lab-exercise-1/BunradarFeature.java:4]: Thank you for the edit.

File: lab-exercise-1/BunradarFeature.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+  public class BunradarFeature {
+      public void feature() {
+          // this is a new feature
+          int x = 0;
+          x = x+1;
```
Comment (ACitronella) [inline lab-exercise-1/BunradarFeature.java:5]: outstanding change!

Comment (Pakin-aw) [inline lab-exercise-1/BunradarFeature.java:5]: Nice implementation.

File: lab-exercise-1/BunradarFeature.java [hunk 6]
```diff
@@ -0,0 +1,7 @@
+  public class BunradarFeature {
+      public void feature() {
```
Comment (RolinAnn) [inline lab-exercise-1/BunradarFeature.java:2]: Nice feature.

File: lab-exercise-1/BunradarFeature.java (hunk 5 above)
Comment (sorrymaker1130) [inline lab-exercise-1/BunradarFeature.java:5]: Add the part successful.

Comment (RolinAnn) [inline lab-exercise-1/BunradarFeature.java:5]: Thank you for adding this part.

File: lab-exercise-1/BunradarFeature.java [hunk 7]
```diff
@@ -0,0 +1,7 @@
+  public class BunradarFeature {
```
Comment (BunradarCH) [inline lab-exercise-1/BunradarFeature.java:1]: good

**Pull request merged**